src/eei.cpp

```diff
@@ -516,7 +564,7 @@ namespace HeraVM {
         addressOffset << " " <<
         valueOffset << " " <<
         dataOffset << " " <<
-        dataLength << dec << "\n";
+        dataLength << " " << dec << "\n";
```

Comment:
I think this is not needed. `dec` as misleading it is (perhaps should used explicit `std::dec` here), is switching to decimal display mode for numbers

Reply:
This change is leftover from an old version of Hera. May need to make a fresh PR, so that unintended changes like these do not arise.